parquet-column/src/main/java/org/apache/parquet/column/statistics/Statistics.java

```diff
@@ -150,26 +150,26 @@ public Float16Builder(PrimitiveType type) {
 
     @Override
     public Statistics<?> build() {
-      Float16Statistics stats = (Float16Statistics) super.build();
+      BinaryStatistics stats = (BinaryStatistics) super.build();
       if (stats.hasNonNullValue()) {
         Binary bMin = stats.genericGetMin();
         Binary bMax = stats.genericGetMax();
         short min = bMin.get2BytesLittleEndian();
         short max = bMax.get2BytesLittleEndian();
         // Drop min/max values in case of NaN as the sorting order of values is undefined for this case
         if (Float16.isNaN(min) || Float16.isNaN(max)) {
-          bMin = Binary.fromConstantByteArray(Float16.POSITIVE_ZERO_BYTES_LITTLE_ENDIAN);
-          bMax = Binary.fromConstantByteArray(Float16.POSITIVE_ZERO_BYTES_LITTLE_ENDIAN);
+          bMin = Binary.fromConstantByteArray(new byte[] {0x00, 0x00});
+          bMax = Binary.fromConstantByteArray(new byte[] {0x00, (byte) 0x80});
```

Comment:
I think, it would be better to use constants with the proper naming. You might even use constants of `Binary` directly.

Reply:
updated, thanks!